firebase-app-distribution/src/test/java/com/google/firebase/appdistribution/FirebaseAppDistributionTest.java

```diff
@@ -271,6 +242,8 @@ public void updateToLatestRelease_whenNewAabReleaseAvailable_showsUpdateDialog()
     AppDistributionReleaseInternal latestRelease = TEST_RELEASE_NEWER_AAB_INTERNAL.build();
     when(mockCheckForUpdateClient.checkForUpdate()).thenReturn(Tasks.forResult(latestRelease));
     firebaseAppDistribution.setCachedLatestRelease(latestRelease);
+    when(mockUpdateAppClient.getUpdateTask(latestRelease, activity))
+        .thenReturn(new UpdateTaskImpl(Tasks.forResult(any())));
```

Comment:
I don't think this should use an argument matcher in the return of the mock. I think this should return a Tasks.ForResult(UpdatTaskImpl(Tasks.forResult(<UpdateStatus>))